get-involved.md

```diff
@@ -3,23 +3,36 @@ layout: page
-    - Create an account and join [our Matrix chat](https://chat.tomesh.net/#/room/#tomesh:tomesh.net)
-    - Alternatively, if you are using the [Element mobile or desktop app](https://element.io/), select _Create Account_, select _Other_ as the homeserver, and enter `https://matrix.tomesh.net` as the homeserver URL. If you already have an account select _Sign In_, select _Change_ and enter our homeserver URL above.
-    - You can also join our chat through a{% include icon-slack.html %} if you really love Slack, note that private messaging only currently works between other Slack users.
+```
+How would you like to get involved? For example, is there a particular working group you are interested in joining?
 
-3. Join one of our [events]({{ site.baseurl }}/events)
-    - We host in-person and virtual meetups and workshops
+...
+How did you find out about Toronto Mesh?
 
-## Funding the Project
+...
+Do you already know someone at Toronto Mesh? If so, let us know who they are!
 
-Toronto Mesh consists of a collective of people and does not currently hold legal status as an organization, nor do we have a bank account. Workshops, conferences, and deployments are generally paid out-of-pocket by the people organizing that particular initiative or through grants.
+...
+Would you like to maintain a Toronto Community Network node? Let us know and we'll get back to you with more information.
 
-Grant applications are applied for in partnership with non-profit organizations, and each application is agreed upon using [our general method of establishing consensus](https://github.com/tomeshnet/documents/blob/master/governance/coordination-structure.md#decision-making), either at meetings or virtually on [#governance:tomesh.net](https://chat.tomesh.net/#/room/#governance:tomesh.net). We keep [a listing of all grants and sources of institutional funding](https://github.com/tomeshnet/documents/tree/master/governance/funding.md) that involve Toronto Mesh as a partner.
+...
+Do you have any other questions for us?
 
-If you would like to contribute to the Toronto Mesh project as an organization, or have any questions regarding our finances, please email [funding@tomesh.net](mailto:funding@tomesh.net).
+...
+Preferred name:
+Other contact information (e.g. Signal, Matrix):
+```
+<br />
+
+## Chat
+
+Say hi on our chat {% include icon-matrix.html %}
+- Create an account and join [our Matrix chat](https://chat.tomesh.net/#/room/#tomesh:tomesh.net)
+- Alternatively, if you are using the [Element mobile or desktop app](https://element.io/), select _Create Account_, select _Other_ as the homeserver, and enter `https://matrix.tomesh.net` as the homeserver URL. If you already have an account select _Sign In_, select _Change_ and enter our homeserver URL above.
+- You can also join our chat through a{% include icon-slack.html %} if you really love Slack, note that private messaging only currently works between other Slack users.
```

Comment:
```suggestion
- You can also join our chat through {% include icon-slack.html %} if you really love Slack. Note that the Matrix-Slack bridge only supports basic features.
```

Reply:
This will read as "You can also join our chat through bridged Slack workspace".  "our bridged Slack workspace" is perhaps better?  Either way we shouldn't straight up remove the "a" without replacing it or rewording icon-slack.html

Reply:
I'm lost do what you need

Reply:
You removed the "a" before the slack include.  Change it to "our" or leave it as is.